### PR TITLE
fix(ui5-view-settings-dialog): adjust scrollbar behavior 

### PR DESCRIPTION
**Pull request by unazko**
Problem description:
The ui5-dialog's content could have a redundant
horizontal scrollbar on Safari and Firefox when
there is a vertical scrollbar placed as an overlay.

Solution:
A right margin is applied to the ui5-view-settings-dialog
content and ui5-date-picker content.

Fixes: #4724
Fixes: #4725
Fixes: https://github.com/SAP/ui5-webcomponents/issues/4860

### CODE REVIEW

File: packages/main/src/themes/PopupsCommon.css [hunk 1]
```diff
@@ -55,6 +55,7 @@
 
 .ui5-popup-content {
     overflow: auto;
+    scrollbar-gutter: stable;
```
Comment (ilhan007) [inline packages/main/src/themes/PopupsCommon.css:58]: Is it supported on Safari
https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter

There is some related issue, we need to look at it
https://bugs.webkit.org/show_bug.cgi?id=167335

Comment (unazko) [reply packages/main/src/themes/PopupsCommon.css]: You are right. The property isn't currently supported for Safari.

**Comment by ilhan007** [comment]
Hello @unazko previously it was a central fix in the Popover and we did a review, it turned out not to be the right direction.
Now that the change is related to the VSD and Calendar only, you can request a review from team @SAP/ui5-webcomponents-topic-b as usual.

**Review by a-z-ivanov — COMMENTED** [review]
I cannot reproduce the horizontal scrollbar in the ViewSettingsDialog - Firefox on Mac. Is the change still needed?

**Comment by ilhan007** [comment]
Hello @unazko could you check if this issue https://github.com/SAP/ui5-webcomponents/issues/4860 is also related to the fix in this PR

**Review by tsanislavgatev — APPROVED** [review]
I've checked it on safari, and no scrollbars were visible. As Alex previoursly approved it, I am giving approve as well.

**Comment by unazko** [comment]
Hello colleagues,

The following issue:
https://github.com/SAP/ui5-webcomponents/issues/4860 will be also fixed via the pull request.
After contacting @PetyaMarkovaBogdanova I was able to reproduce it on Chrome browser and windows as well
and confirmed that it will be fixed via the given change.

Best Regards,
Boyan Rakilovski

**Review by ilhan007 — APPROVED** [review]

**Pull request merged**